### PR TITLE
NMS-12917: Update Minion installation documentation

### PR DESCRIPTION
**Pull request by Bonrob2**
### All Contributors

* [ x] Have you read and followed our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [x ] Have you [made an issue in the OpenNMS issue tracker](https://issues.opennms.org)?<br>If so, you should:
  1. update the title of this PR to be of the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
* [ x] Have you made a comment in that issue which points back to this PR?
* [x ] Have you updated the JIRA link at the bottom of this comment to link to your issue?
* [ ] If this is a new or updated feature, is there documentation for the new behavior?
* [ ] If this is new code, are there unit and/or integration tests?
* [ ] If this PR targets a `foundation-*` branch, does it avoid changing files in `$OPENNMS_HOME/etc/`?

### Pull Request Process

One or more reviewers should be assigned to each PR.

If you know that a particular person is subject matter expert in the area your PR affects, feel free to assign one or more reviewers when you create this PR, otherwise reviewers will be assigned for you.

If you have made additions or changes to the documentation, or if you _need_ documentation for these code changes, please make sure a technical writer has looked it over.

Once the reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the Order of the Green Polo) you are welcome to merge the PR.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-12917

### CODE REVIEW

File: opennms-doc/guide-install/src/asciidoc/text/monitor-location-minion/debian/step-2-starting-minion.adoc [hunk 1]
```diff
@@ -4,24 +4,25 @@
...
+[subs=+quotes]
 ----
 ssh -p 8201 admin@localhost
 
 config:edit org.opennms.minion.controller
-config:property-set location #Office-Pittsboro
-config:property-set http-url #http://opennms-fqdn:8980/opennms
-config:property-set broker-url failover:#tcp://opennms-fqdn:61616
+config:property-set location [red]#Office-Pittsboro#
+sudo config:property-set http-url [red]#http://opennms-fqdn:8980/opennms#
```
Comment (GanimanSwift) [inline opennms-doc/guide-install/src/asciidoc/text/monitor-location-minion/debian/step-2-starting-minion.adoc:24]: I don't think a sudo belongs here. This is karaf command line, not bash.

File: opennms-doc/guide-install/src/asciidoc/text/monitor-location-minion/debian/step-2-starting-minion.adoc [hunk 2]
```diff
@@ -30,9 +31,10 @@ TIP: Include the `failover:` portion of the broker URL to allow the Minion to re
 
 .Configure the credentials to use when communicating with {opennms-product-name} and exit Karaf shell
 [source]
+[subs=+quotes]
 ----
-opennms:scv-set opennms.http #minion user name minion password
-opennms:scv-set opennms.broker #minion username minion password
+opennms:scv-set opennms.http [red]#minion user name minion password#
```
Comment (GanimanSwift) [inline opennms-doc/guide-install/src/asciidoc/text/monitor-location-minion/debian/step-2-starting-minion.adoc:36]: I think "username" should be one word everywhere. It is one argument of this karaf command, not two, so I think having it as "user name" is incorrect.

**Review by GanimanSwift — CHANGES_REQUESTED** [review]
I've made a couple of notes. Please double check them.

**Review by Bonrob2 — COMMENTED** [review]
updated as per review

**Comment by opennms-bamboo** [comment]
Welcome CI/CD System (cicd-system@opennms.com).

We would like to thank you for your contribution.

I could not find a signed OpenNMS Contributor Agreement (OCA) assigned to the username CI/CD System (cicd-system@opennms.com).
In order to maintain some order with respect to the copyright to the code,
every contributor is asked to sign a "Contributor Agreement".

The OpenNMS Contributor Agreement can be downloaded [here](https://wiki.opennms.org/wiki/images/6/6b/ContributorAgreement.pdf).
Please send a signed copy to oca@opennms.org.

If you need more information, please visit https://wiki.opennms.org/wiki/Contributor_Agreement

**Comment by indigo423** [comment]
@Bonrob2 I bet we introduced bugs by introducing the sudo commands which changes the whole install procedure to run as non-root user which we never did or tested before. All other install instructions are described from a root user. So there are definitely here.

**Comment by Bonrob2** [comment]
@ronny-trommer Ok, I can roll this back. In reviewing the docs after the first update, Chris suggested adding the sudo commands, which is what I did here. We do say that all procedures must be run as root. Let me know if we should roll it back.

**Comment by indigo423** [comment]
> @ronny-trommer Ok, I can roll this back. In reviewing the docs after the first update, Chris suggested adding the sudo commands, which is what I did here. We do say that all procedures must be run as root. Let me know if we should roll it back.

To be perfectly honest, I would vote to roll it back and if it's a requirement to provide docs to install as an unprivileged non-root user we should migrate the whole installation guides and verify they work.

**Comment by Bonrob2** [comment]
Sounds good. Thanks.

**Pull request merged**